### ISSUE TITLE
elements in ibm-dropdown-list cannot be read by JAWS

### ISSUE THREAD

**Issue by lj7720** [IBM/carbon-components-angular#1747]
## Detailed description

> Describe in detail the issue you're having.

When focus moves to ibm-dropdown-list, JAWS cannot read the elements of both label or checkbox in the list.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

ibm-dropdown-list

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?

Firefox 85.0.2

> What version of the Carbon Design System are you using?

3.54.2

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

ADDI

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/74904663/110089922-a7b18d00-7dd1-11eb-95a2-0f5034cb2075.png)
![image](https://user-images.githubusercontent.com/74904663/110090064-d465a480-7dd1-11eb-98f8-4f6b4f1f750e.png)

* Notes

**Comment by lj7720** [comment]
hi, @cal-smith This issue was still pending and it will affect our next accessibility compliance report, would you please arrange to resolve asap? Thanks.
@stanislavgeorgiev

**Comment by stanislavgeorgiev** [comment]
@ninja511 The ADDI Extensions have been updated to the 4.x branch. Can you please do a test if this is still an issue?

**Comment by cal-smith** [comment]
Just checked, and this does appear to still be an issue ... Strange 🤔

**Comment by cal-smith** [comment]
Doing some testing and it appears that it recognizes the items in the listbox, but it doesn't read them out (for whatever reason)

**Comment by stanislavgeorgiev** [comment]
Yes, the AVT team confirmed it still as an issue. I just installed JAWS on my Windows VM and will work on this today

**Comment by cal-smith** [comment]
> Yes, the AVT team confirmed it still as an issue. I just installed JAWS on my Windows VM and will work on this today

Whoops, just saw this @stanislavgeorgiev, I had done some digging on this and we had to totally change how we were approaching focus states for the listbox ... I've added it to my misc fixes PR

**Comment by stanislavgeorgiev** [comment]
oh wow, awesome!
Thanks @cal-smith , the PR looks like quite a bit is going on in there. 
Would it be merged today?

**Comment by cal-smith** [comment]
That's the plan 👍 as long as there's no show stoppers we should be good to go

**Comment by stanislavgeorgiev** [comment]
Jie confirmed in the PR preview that JAWS is now working ok 🥳

**Comment by cal-smith** [comment]
Awesome!

**Comment by stanislavgeorgiev** [comment]
🥳
